src/js/view/controls/components/timeslider.js

```diff
@@ -69,12 +69,14 @@ function reasonInteraction() {
 }
 
 class TimeSlider extends Slider {
-    constructor(_model, _api) {
+    constructor(_model, _api, _playerContainer) {
         super('jw-slider-time', 'horizontal');
 
         this._model = _model;
         this._api = _api;
 
+        this.accessibilityContainer = _playerContainer.querySelector('.jw-hidden-accessibility');
```

Comment:
Pass in `accessibilityContainer` not `_playerContainer` on setup. There is no use for `_playerContainer` other than this query which should be done externally.